rest-api/dao/biobank_order_dao.py

```diff
@@ -94,6 +94,8 @@ def _order_as_dict(self, order):
     return result
 
   def insert_with_session(self, session, obj):
+    if obj.version == None:
+      obj.version = 1
```

Comment:
I would make this unconditional. Clients shouldn't be able to dictate the version.

You're also going to need some logic to bump the version during updates. (Though you could add this later.)